sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
+	"Resources": {
+		"ServiceRoot": {
+			"PropertyRequirements": {
+				"Chassis": {
```

Comment:
From the `sushy` source code, I believe this property's `ReadRequirement` is `Recommended`. According to the specification, "The default value, or if no `ReadRequirement` is present, is `Mandatory`."  Therefore, I believe `ReadRequirement` must be explicitly provided.